aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 5.3.1
+version: 5.3.2
```

```suggestion
version: 5.4.0
```
This is a new feature, according to semver we should bump minor version.

---

makes sense, thanks. Changed